src/main/java/core/service/WriterServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.service;
```

Comment:
```suggestion
package core.service.impl;
```
let's put implementations into impl package